src/System.Runtime.Numerics/ref/System.Runtime.Numerics.cs

```diff
@@ -13,7 +13,7 @@ namespace System.Numerics
     {
         [System.CLSCompliantAttribute(false)]
         public BigInteger(byte[] value) { throw null; }
-        public BigInteger(System.ReadOnlySpan<byte> value) { throw null; }
+        public BigInteger(System.ReadOnlySpan<byte> value, bool isUnsigned=false, bool isBigEndian=false) { throw null; }
```

Comment:
Do we normally use the `is` prefix for bool parameter names? `Is` is more common for bool members (properties/methods), but parameters? I would think these would be named `unsigned` and `bigEndian`.

There's some prior art: `UnicodeEncoding` and `UTF32Encoding` have the following constructors:

```c#
public UnicodeEncoding(bool bigEndian, bool byteOrderMark);
public UTF32Encoding(bool bigEndian, bool byteOrderMark);
```

The parameters are named `bigEndian` and `byteOrderMark`, not `isBigEndian` and `hasByteOrderMark`.

Reply:
Prior art for pretty much exact situation is: `public Decimal(int lo, int mid, int hi, bool isNegative, byte scale)`

Reply:
@terrajobst I don't have a particular opinion about (`isUnsigned`, `isBigEndian`) vs (`unsigned`, `bigEndian`), but the `is` prefix didn't come up at all in the API review.